frontend/src/components/pages/map.js

```diff
@@ -0,0 +1,9 @@
+import React, { useEffect, useRef, useState } from 'react';
+import { useLocation, Link } from 'react-router-dom';
+import { BrowserRouter as Router, Route, Routes, useNavigate } from 'react-router-dom';
+
+export default function MapItem(prop) {
+    return (
+        <div>Hello from MapItem</div>
```

Comment:
Is this a debug page?

Reply:
That was a mistake, file's not supposed to be there